examples/llm/src/mosaic_gpt.py

```diff
@@ -421,10 +421,12 @@ def forward(self,
                                     seq_len=S,
                                     key_padding_mask=key_padding_mask,
                                     dtype=x.dtype)
+        if self.cfg.attn_impl == 'flash':
+            key_padding_mask = torch.ones_like(input_ids, dtype=torch.bool)
+        if self.cfg.attn_impl == 'triton':
```

Comment:
```suggestion
        elif self.cfg.attn_impl == 'triton':
```
for clarity that these are mutually exclusive code paths

Reply:
now it needs the elif, otherwise will ignore the setting in the flash conditional and hit the else